src/Security/Authorization/Core/src/AuthorizeAttribute.cs

```diff
@@ -39,4 +41,12 @@ public AuthorizeAttribute(string policy)
     /// Gets or sets a comma delimited list of schemes from which user information is constructed.
     /// </summary>
     public string? AuthenticationSchemes { get; set; }
+
+    /// <inheritdoc/>>
+    public override string ToString()
+    {
+        const string NullValue = "(null)";
+
+        return $"Policy: {Policy ?? NullValue}, Roles: {Roles ?? NullValue}, AuthenticationSchemes: {AuthenticationSchemes ?? NullValue}";
```

Comment:
```suggestion
        return $"Authorization Policy: {Policy ?? NullValue}, Roles: {Roles ?? NullValue}, AuthenticationSchemes: {AuthenticationSchemes ?? NullValue}";
```

Reply:
Could this be more dynamic and only show the fields that are set? It's very common to only use one of these fields.
"Authorization Policy: MyAuthPolicy"
"Authorization Roles: Role1, Role2"
"AuthenticationSchemes: Bearer"

Reply:
That makes the implementation much more complicated.

I would want a helper method to do this. You're starting me down the path of writing a serializer 😱

Reply:
Ah, you wouldn't know anything about that...